packages/gatsby/content/getting-started/migration.md

```diff
@@ -1,7 +1,7 @@
 ---
-category: advanced
-path: /advanced/migration
-title: "Migration"
+category: getting-started
+path: /getting-started/migration
```

Comment:
Can you also add a redirect on the `_redirects` file? Otherwise we'll break the links to the migration guide.